### PR TITLE
Release v0.1.336

### PR DESCRIPTION
**Pull request by paradigm**
Update model to v0.0.280, adding account_group_id to role_binding

### CODE REVIEW

**Comment by rawsyntax** [comment]
Typically the update is done in a separate pull request and the release is done by itself. For example:

https://github.com/openshift-online/ocm-sdk-go/pull/745
https://github.com/openshift-online/ocm-sdk-go/pull/746

File: accountsmgmt/v1/access_token_auth_builder.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ limitations under the License.
 package v1 // github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1
 
 // AccessTokenAuthBuilder contains the data and logic needed to build 'access_token_auth' objects.
+//
+//
```
Comment (rawsyntax) [inline accountsmgmt/v1/access_token_auth_builder.go:24]: I'm not sure what is generating these changes, but it looks like too many files are touched. We'll need to figure out how to fix this before merging.

@gdbranco please take a look

Comment (gdbranco) [reply accountsmgmt/v1/access_token_auth_builder.go:24]: Need >= go@1.19 higher and >= goimports@0.4.0 if I'm not mistaken

`make fmt` might also resolve it, but with those versions it should work with make generate
@vkareh do you remember the command for fmt?

Comment (vkareh) [reply accountsmgmt/v1/access_token_auth_builder.go:24]: I think just `make fmt` should clean them up. I get those changes every time until I remember to format it

**Comment by renan-campos** [comment]
Closing this PR, since we're a few releases ahead of this by this point.

**Pull request closed without merging**